tip-wlan/charts/kafka/values.yaml

```diff
@@ -353,3 +353,8 @@ zookeeper:
   #       labelSelector:
   #         matchLabels:
   #           release: zookeeper
+
+creds:
```

Comment:
While we are converting `cassandra-application.conf` and Postgres datasource (`datasource.properties`) to configMaps, what are your views on `admin-client.properties` file under Kafka? Should that get converted into ConfigMap as well

Reply:
thanks! updated it too